boards/wiznet/w5500_evb_pico2/w5500_evb_pico2_rp2350a_m33_defconfig

```diff
@@ -0,0 +1,11 @@
+CONFIG_SYS_CLOCK_HW_CYCLES_PER_SEC=150000000
```

Comment:
should be set in the soc `Kconfig` file from the dts property (can be fixed in another PR)

Reply:
Moved it into Kconfig.defconfig.